networking/build.gradle

```diff
@@ -21,8 +23,13 @@ dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
 
     // Wolmo
-    compile 'com.github.Wolox:wolmo-core-android:master'
+    compile 'com.github.Wolox:wolmo-core-android:v0.5.0'
 
     // Third party
     compile 'com.squareup.retrofit2:retrofit:2.2.0'
+    compile 'com.squareup.retrofit2:converter-gson:2.2.0'
+    compile 'com.squareup.okhttp3:okhttp-urlconnection:3.7.0'
+    compile 'com.squareup.okhttp3:okhttp:3.7.0'
+    compile 'com.squareup.okhttp3:logging-interceptor:3.7.0'
+    compile 'joda-time:joda-time:2.9.9'
```

Comment:
Makes sense to get rid of `Joda Time` soon, but I would recommend making that modification in a future PR, since it's not a mission critical fix, just a nice to have. 
I'll add a Github issue for that so we can take a look into that change (we will probably have to get rid of the Joda Time in Wolmo Core as well).